src/temp.cpp

```diff
@@ -0,0 +1,10 @@
+#include "temp.h"
+#include <math.h>
+#include <stdio.h>
+using namespace std;
```

Comment:
This is not recommended since it increases chance for accidental conflicts. Since you guys are only using it for log, remove